CMakeLists.txt

```
@@ -1078,6 +1078,16 @@ if(SDLIMAGE_INSTALL)
             REVISION "${SDLIMAGE_REVISION}"
         )
     endif()
+
+
+    ##### Uninstall target #####
+
+    if(NOT TARGET uninstall)
+        configure_file(cmake/cmake_uninstall.cmake.in cmake_uninstall.cmake IMMEDIATE @ONLY)
+
+        add_custom_target(uninstall
+            COMMAND ${CMAKE_COMMAND} -P "${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake")
+    endif()
```

I think it only makes sense to create an `uninstall` target when "our project" (=SDL_image) is the main project.
```suggestion

    if(SDLIMAGE_ROOTPROJECT)
        # Uninstall target
        configure_file(cmake/cmake_uninstall.cmake.in cmake_uninstall.cmake IMMEDIATE @ONLY)
        add_custom_target(uninstall
            COMMAND ${CMAKE_COMMAND} -P "${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake")
    endif()
```

---

I disagree, imo uninstall should not depend on it being the main project but it having a install target. If there is an option to install there should be an option to uninstall.

But no hard feelings my use case is the standalone project so works for me. Applied your change.